templates/security/fips.html

```diff
@@ -0,0 +1,209 @@
+          </td>
+          <td class="u-align--center">
+            <p>Merged with OpenSSL</p>
+          </td>
+        </tr>
+        <tr>
+          <th>OpenSSL</th>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3725">#3725</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3622">#3622</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/Modules-In-Process-List">Under validation</a>
+          </td>
+        </tr>
+        <tr>
+          <th>libgcrypt</th>
+          <td class="u-align--center" aria-label="None">–</td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3748">#3748</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3902">#3902</a>
+          </td>
+        </tr>
+        <tr>
+          <th>StrongSwan</th>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2978">#2978</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/3648">#3648</a>
+          </td>
+          <td class="u-align--center">
+            <a class="p-link--external" href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/Modules-In-Process-List">Under validation</a>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+    <a class="p-button--positive js-invoke-modal" href="/support/contact-us">Access the FIPS140-2 validated modules</a>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h3>Canonical patches vulnerabilities</h3>
+    <p>Each FIPS 140-2 certificate is valid for 5 years, however vulnerabilities happen, and it is our intention to publish fixed packages quickly, irrespective of their certification status. We therefore provide two alternative options. An option to remain with the certified cryptographic packages (called the 'fips' option), and an option to use the certified packages but include security fixes (called the 'fips-updates' option) when available. Check <a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/fips">our security pages on how to enable these options</a>.</p>
```

Comment:
```suggestion
    <p>Each FIPS 140-2 certificate is valid for 5 years, however vulnerabilities happen, and it is our intention to publish fixed packages quickly, irrespective of their certification status. We therefore provide two alternative options. An option to remain with the certified cryptographic packages (called the 'fips' option), and an option to use the certified packages but include security fixes (called the 'fips-updates' option) when available. Check <a class="p-link--external" href="https://security-certs.docs.ubuntu.com/en/fips">our security pages on how to enable these options</a>.</p>
```
Change diagonal apostrophes to straight ones